.github/templates/windows_binary_build_workflow.yml.j2

```diff
@@ -92,6 +97,8 @@ jobs:
 {%- else %}
     runs-on: windows.8xlarge.nvidia.gpu.nonephemeral
 {%- endif %}
+{%- elif config["gpu_arch_type"] == "xpu" %}
```

Comment:
For current stage, we can try to use `windows.4xlarge.nonephemeral` to do some sannity test, eg. `import torch`

Reply:
Not sure what machine do we use to test. Put it here since Nvidia did.

Reply:
Yes, understand. Due to our MTL runner is very limited. Let's try to use `windows.4xlarge.nonephemeral` firstly.